src/actions/index.js

```diff
@@ -29,6 +32,19 @@ export const getAllTransactions = () => dispatch => {
 		});
 };
 
+export const clearTransactions = () => {
+	return({
```

Comment:
Immediately returning functions can be collapsed when using fat arrows

Reply:
Done.